### ISSUE TITLE
Why finetune on the validation set?

### ISSUE THREAD

**Issue by zhiweige** [yihui-he/channel-pruning#72]
Hi, Thanks for your meaningful work.
I have a question about the finetuning process.
When I finetune, I noticed that the data_param is
data_param {
    source: "/data/xxxx/ILSVRC12_val_lmdb"
    batch_size: 32
    backend: LMDB
}
Is it a typo? Or we need to finetune on the validation set?
Thanks.

**Comment by yihui-he** [comment]
where did you see it? I can't find it in my repo.

**Comment by zhiweige** [comment]
okay, thanks!
